NewsApi/analyzer.py

```diff
@@ -0,0 +1,111 @@
+"""
+    This contains sentiment analysis models
+"""
+import json
+import pandas as pd
+import nltk
+nltk.download('vader_lexicon')
+
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+from flask import request
+from newsapi import NewsApiClient
+from NewsApi.api_key import api_key
+from sqlalchemy import null, exc
+
+
+class DataProcessor:
+    @staticmethod
+    def data_processing():
+        keyword = request.json['keyword']
```

Comment:
Where is this `request` coming from? (Yes, it is from `flask` but we don't know what value it will hold when this function is called).

You could pass the `request` (or even better rather the data you need) to this function.